### PR TITLE
Simplify action descriptions

### PR DESCRIPTION
**Pull request by bramkragten**
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/data/script_i18n.ts [hunk 1]
```diff
@@ -174,54 +174,25 @@ export const describeAction = <T extends ActionType>(
...
-    } then ${ensureArray(config.then).map((thenAction) =>
-      describeAction(hass, thenAction)
-    )}${
-      config.else
-        ? ` else ${ensureArray(config.else).map((elseAction) =>
-            describeAction(hass, elseAction)
-          )}`
-        : ""
+    return `Perform an action based on a condition${
+      config.else ? ", else perform other action" : ""
```
Comment (balloob) [inline src/data/script_i18n.ts:178]: Should we include the else part?

**Review by balloob — DISMISSED** [review]

File: src/data/script_i18n.ts (hunk 1 above)
Comment (balloob) [reply src/data/script_i18n.ts:178]:
```suggestion
      config.else ? " (or else!)" : ""
```

**Comment by frenck** [comment]
Can we as translation keys for these things?

**Comment by bramkragten** [comment]
> Can we as translation keys for these things?

add you mean? Yes, but not before we nail the english ones, as it would cause a lot of duplicate work and breaking changes

**Review by balloob — APPROVED** [review]

**Pull request merged**